dymos/run_problem.py

```diff
@@ -75,8 +75,7 @@ def run_problem(problem, refine_method='hp', refine_iteration_limit=0, run_drive
     if solution_record_file not in [rec._filepath for rec in iter(problem._rec_mgr)]:
         recorder = om.SqliteRecorder(solution_record_file)
         problem.add_recorder(recorder)
-        # record_inputs is needed to capture potential input parameters that aren't connected
```

Comment:
Why did you remove the inputs from the case recorder?

I mostly ask this because today, I read in a case db from an Aviary run and did a list_inputs on it.

Reply:
It should be unnecessary now since everything is ultimately an output.  This was also a fast way around an openmdao bug in set val when it attempts to set a remote input.  There is an issue up for that, but I thought it made sense to stop recording redundant data